code/game/objects/effects/spiderwebs.dm

```diff
@@ -105,13 +105,11 @@
 
 /obj/structure/spider/stickyweb/genetic/Initialize(mapload, allowedmob)
 	. = ..()
+	allowed_mob = allowedmob
+	RegisterSignal(allowed_mob, COMSIG_QDELETING, PROC_REF(on_allowed_mob_delete))
```

Comment:
I am very curious how tests are passing, given no mob is passed, this should be runtiming. 

But also: I feel as if we should just use weakrefs for this. The signal feels like overkill.